pandas/tests/frame/methods/test_astype.py

```diff
@@ -767,13 +764,9 @@ def test_astype_dt64_to_string(
         if frame_or_series is DataFrame:
             item = item.iloc[0]
         if using_infer_string:
-            assert item is np.nan
-        else:
             assert item is pd.NA
-
-        # For non-NA values, we should match what we get for non-EA str
-        alt = obj.astype(str)
```

Comment:
Maybe repeat the above also with `dta.astype("str")`, so we test the default string dtype as well

Reply:
Meant to leave a comment on this. It's not visible in the diff but there is already a `tm.assert_frame_equal` call a few lines up from this. Is there any expected value calling that and then calling it with a slice?

Reply:
I meant to repeat the `expected = frame_or_series(dta.astype("string"))` as `expected = frame_or_series(dta.astype("str"))` (to test both the NA and NaN variant)